zipkin-sampler/src/test/scala/com/twitter/zipkin/sampler/RequestRateCheckTest.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2012 Twitter Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.zipkin.sampler
+
+import com.twitter.conversions.time._
+import com.twitter.finagle.stats.NullStatsReceiver
+import com.twitter.util.{MockTimer, Time, Var}
+import org.scalatest.FunSuite
+
+class RequestRateCheckTest extends FunSuite {
```

Comment:
this file was incorrectly named AdaptiveSamplerTest before. It is just a rename, but git didn't understand.